tools/c7n_azure/c7n_azure/resources/cosmos_db.py

```diff
@@ -483,7 +483,7 @@ def _process_account_set(self, resources, account_client):
                     tag_name, parent_account))
 
         except Exception as e:
-            log.warn(e)
+            log.warning(e)
```

Comment:
I'm going to save additional log refactoring for a different pull request (coming soon though) as there are many many instances.  I just fixed this because it was throwing a deprecation warning.